pkg/csource/build.go

```diff
@@ -44,7 +44,11 @@ func build(target *prog.Target, src []byte, file string) (string, error) {
 		"-DGOARCH_" + target.Arch + "=1",
 	}
 	if file == "" {
-		flags = append(flags, "-x", "c", "-")
+		language := "c"
+		if target.OS == "fuchsia" {
+			language = "c++"
```

Comment:
Why is this necessary?

Reply:
I added this because FIDL types are defined in C++, (e.g., fuchsia-process.fidl.tables.cc)  and "syz-prog2c -build" was failing to build a test program generated by syz-mutate.

Reply:
Is this still needed? If not, please remove.